Makefile

```diff
@@ -40,51 +40,131 @@ buildnocache:
 	./ci_cd/version.sh
 	@docker-compose build --no-cache --pull
 
+build:
+	./ci_cd/version.sh
+	@docker-compose build
+
 up:
 	docker-compose up -d
 
+logs:
+	@docker-compose logs -f $(API_SERVICE)
+
 start:
 	@make up
 
+restart_api:
+	@docker-compose restart api_service
+
+# -----------------
+# DB actions
+# -----------------
 dbbackup:
-	@docker-compose exec --user=$(CURRENT_UID) $(API_SERVICE) python manage.py dbbackup local
+ifdef target
+	@docker-compose run \
+		--rm \
+		--name api_dbbackup \
+		-e AWS_ACCESS_KEY_ID=${OLD_AWS_ACCESS_KEY_ID} \
+		-e AWS_SECRET_ACCESS_KEY=${OLD_AWS_SECRET_ACCESS_KEY} \
+		-e PGPASSWORD=postgres \
+		--entrypoint python \
+		--user=$(CURRENT_UID) \
+		$(API_SERVICE) \
+		manage.py dbbackup $(target)
+else
+	@echo "Please specify a target. \nie: make dbbackup target=local"
+endif
+
+db:
+	@echo "Test"
 
 dbrestore:
-	@docker-compose exec --user=$(CURRENT_UID) $(API_SERVICE) python manage.py dbrestore local
+ifdef target
+	@docker-compose run \
+		--rm \
+		--name api_dbrestore \
+		-e AWS_ACCESS_KEY_ID=${OLD_AWS_ACCESS_KEY_ID} \
+		-e AWS_SECRET_ACCESS_KEY=${OLD_AWS_SECRET_ACCESS_KEY} \
+		-e PGPASSWORD=postgres \
+		--entrypoint python \
+		--user=$(CURRENT_UID) \
+		$(API_SERVICE) \
+		manage.py dbrestore $(target)
+else
+	@echo "Please specify a target. \nie: make dbrestore target=local"
+endif
 
 migrate:
-	@docker-compose exec --user=$(CURRENT_UID) $(API_SERVICE) python manage.py migrate
-
+	@docker-compose run \
+		--rm \
+		--name api_migrate \
+		--entrypoint python \
+		--user=$(CURRENT_UID) \
+		$(API_SERVICE) \
```

Comment:
I now get why we call `run` for `dbbackup` and `dbrestore` and even `test`; but for `migrate` shouldn't we be acting on `api_runserver`, because of third-party migration files?

Reply:
migrate only applies the changes to the DB